### PR TITLE
Update SLEAP guide for newest module version

### PR DESCRIPTION
**Pull request by niksirbi**
## Description

Because I installed a new version of SLEAP (`v1.3.3`) as a module on the cluster, I've make some small tweaks to the guide accordingly.

I've also added a note for people with managed Linux desktops (whose job for using the module is much easier).

### CODE REVIEW

File: docs/source/data_analysis/HPC-module-SLEAP.md [hunk 1]
```diff
@@ -12,11 +12,31 @@
...
+The SWC's IT team offers managed desktop computers equipped with a Linux image. These machines are already part of SWC's trusted domain and have direct access to SLURM, the HPC modules, and the SWC filesystem.
+
+If you have access to one of these desktops,
+you can skip the pre-requisite steps.
+You may simply open a terminal, type `module load SLEAP`,
+and start using SLEAP directly, as you would on any local
+Linux machine. All SLEAP commands should work as expected,
+including `sleap-label` for launching the GUI.
+
+That said, you may still want to offload GPU-intensive tasks to other HPC nodes (e.g. because the desktop's GPU is not powerful enough or because you need to run many jobs in parallel). In that case, you may
```
Comment (sfmig) [inline docs/source/data_analysis/HPC-module-SLEAP.md:34]:
```suggestion
That said, you may still want to offload GPU-intensive tasks to an HPC nodes (e.g. because the desktop's GPU is not powerful enough or because you need to run many jobs in parallel). In that case, you may
```
(since their workstations are not HPC nodes right?)

File: docs/source/data_analysis/HPC-module-SLEAP.md [hunk 2]
```diff
@@ -76,19 +98,19 @@ After that, you can follow the [rest of the SLEAP installation guide](https://sl
 
 :::{tab-item} Windows and Linux
 ```{code-block} console
-$ conda create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.1
+$ conda create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.3
```
Comment (sfmig) [inline docs/source/data_analysis/HPC-module-SLEAP.md:101]: It feels a bit of an unnecessary burden to ensure that this section is in sync with SLEAP's docs.

I think we can remove most of it and keep until: 
> We recommend following the official [SLEAP installation guide](https://sleap.ai/installation.html).

I would keep your clarification at the end, I like it, but maybe rephrase a bit to include why:
> To minimise the risk of issues due to incompatibilities between versions, ensure the version of your local installation of SLEAP matches the one you plan to load in the cluster.

File: docs/source/data_analysis/HPC-module-SLEAP.md [hunk 3]
```diff
@@ -114,12 +136,14 @@ $ rsync -avz <LOCAL-DIR> <SWC-USERNAME>@ssh.swc.ucl.ac.uk:/ceph/scratch/neuroinf
 ```
 :::
 
+(sleap-training)=
 ## Model training
-This will consist of two parts - [preparing a training job](#prepare-the-training-job)
-(on your local SLEAP installation) and [running a training job](#run-the-training-job)
+This will consist of two parts - [preparing a training job](prepare-the-training-job)
```
Comment (sfmig) [inline docs/source/data_analysis/HPC-module-SLEAP.md:141]:
```suggestion
This will consist of two parts: [preparing a training job](prepare-the-training-job)
```

File: docs/source/data_analysis/HPC-module-SLEAP.md [hunk 4]
```diff
@@ -346,6 +371,7 @@ If you encounter out-of-memory errors, keep in mind that there two main sources
 - If requesting more memory doesn't help, you can try reducing the size of your SLEAP models. You may tweak the model backbone architecture, or play with *Input scaling*, *Max stride* and *Batch size*. See SLEAP's [documentation](https://sleap.ai/) and [discussion forum](https://github.com/talmolab/sleap/discussions) for more details.
 ```
 
+(evaluate-the-trained-models)=
 ### Evaluate the trained models
```
Comment (sfmig) [inline docs/source/data_analysis/HPC-module-SLEAP.md:375]:
```suggestion
## Model evaluation
```
maybe? That way it matches a bit better the train, eval, inference scheme.

File: docs/source/data_analysis/HPC-module-SLEAP.md [hunk 5]
```diff
@@ -603,10 +630,7 @@ $ exit()
 If you encounter troubles with using the SLEAP module, contact
 Niko Sirmpilatze of the SWC [Neuroinformatics Unit](https://neuroinformatics.dev/).
 
-To completely exit the HPC cluster, you will need to logout of the SSH session  twice:
-```bash
-$ logout
-$ logout
-```
+To completely exit the HPC cluster, you will need to type `exit` or
+`logout` until you are back to your local machine's terminal prompt.
```
Comment (sfmig) [inline docs/source/data_analysis/HPC-module-SLEAP.md:634]:
```suggestion
`logout` until you are back to the terminal prompt of your local machine.
```

**Review by sfmig — APPROVED** [review]
🚀 !

File: docs/source/data_analysis/HPC-module-SLEAP.md (hunk 2 above)
Comment (niksirbi) [reply docs/source/data_analysis/HPC-module-SLEAP.md]: agreed, you're totally right.
